frontend/tests/utils/helpers.ts

```diff
@@ -13,28 +13,26 @@ export async function goToPage(url: string = TEST_URL): Promise<void> {
 }
 
 export async function login(): Promise<void> {
-  goToPage();
+  await goToPage();
 
   expect(process.env.TEST_ACCOUNT_PASS).toBeDefined();
 
-  try {
-    await expect(page).toHaveSelector(getText("My Collections"), {
-      timeout: TIMEOUT_MS,
-    });
-  } catch (error) {
-    const username = TEST_USERNAME;
+  const cookies = await context.cookies();
 
-    await page.click(getText("Log In"));
+  if (cookies.length) return;
```

Comment:
Instead of checking if `Log In` button is in the DOM, which can be flaky, we just check if the browser already has cookies